index.html

```diff
@@ -71,6 +74,23 @@ <h1 class="header">
            
         </section>
     </div>
+
+    <div id="overlay">
+        <div class="close">
+            <img src="./images/close.svg" alt="close image" id="closeMenu">
+        </div>
+        <ul>
+            <li>
+                <a href="">Portfolio</a>
+            </li>
+            <li>
+                <a href="">About</a>
+            </li>
+            <li>
+                <a href="">Contact</a>
+            </li>
+        </ul>
```

Comment:
- [ ] As per these [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=When%20the%20user%20clicks%20(or%20taps)%20any,correct%20part%20of%20the%20page%20is%20displayed.), When I click on any mobile menu options the menu suppose to disappear and take me to the correct part of the page, not refresh the page. So please fix that.

![chrome-capture-2022-9-11](https://user-images.githubusercontent.com/68796232/195067835-fb03ecdc-d683-43f1-90b4-a72df5d6e41d.gif)